### DISCUSSION TITLE
Be compatable with librejs.

### DISCUSSION THREAD

**Discussion by chaoticryptidz** [misskey-dev/misskey#14014] [General]
## Summary

All you need to do is make it so your script that bundles your javascript includes license info for everything.
For more info see: https://www.gnu.org/software/librejs/free-your-javascript.html
This will make it so when a user uses misskey with librejs, they can visit the site without it blocking all javascript.

**Comment by tamaina** [comment]
FOI: http://www.gnu.org/licenses/javascript-labels.html

**Comment by 4ioskd** [comment]
Close the issue?
